lib/main_common.pm

```diff
@@ -501,12 +501,13 @@ sub load_extra_tests() {
             # sysauth test scenarios run in the console
             loadtest "sysauth/sssd";
         }
-        # finished console test and back to desktop
-        loadtest "console/consoletest_finish";
         # kdump is not supported on aarch64, see BSC#990418
         if (!check_var('ARCH', 'aarch64')) {
```

Comment:
I would still enable them even for aarch64. Also the bug reference is wrong bsc#1037407. BSC#990418 is already closed.